CHANGELOG.md

```diff
@@ -17,12 +17,57 @@ Mapbox welcomes participation and contributions from everyone.
+* Fix a bug that `line-trim-offset` input may lose precision via shader calculation. ([#1359](https://github.com/mapbox/mapbox-maps-android/pull/1359))
+* Add mercator scale factor to 3D location puck, so that the 3D puck size won't increase as latitude increases. ([#1350](https://github.com/mapbox/mapbox-maps-android/pull/1350)
+* Fix a crash due to invalid focal point when panning the map. ([#1364](https://github.com/mapbox/mapbox-maps-android/pull/1364))
+* Fix compass is not showing in "edge-to-edge" mode. ([1391](https://github.com/mapbox/mapbox-maps-android/pull/1391))
+* Fix an unexpected request update delay for tiles taken from in-memory cache when minimumTileUpdateInterval is set. ([#1396](https://github.com/mapbox/mapbox-maps-android/pull/1396))
+* Fix a rare bug where some tiles would not show up correctly on globe. ([#1396](https://github.com/mapbox/mapbox-maps-android/pull/1396))
+* Fix a bug where features could be queried without cursor intersecting the globe. ([#1396](https://github.com/mapbox/mapbox-maps-android/pull/1396))
+* Fix view annotations disappearing on the globe. ([#1396](https://github.com/mapbox/mapbox-maps-android/pull/1396))
+* Fix symbols ignoring both collision and placement on the globe. ([#1396](https://github.com/mapbox/mapbox-maps-android/pull/1396))
+* Add anti-aliasing on the globe on low zoom levels. ([#1396](https://github.com/mapbox/mapbox-maps-android/pull/1396))
+* Fix globe controls when map orientation is something else than "north". ([#1396](https://github.com/mapbox/mapbox-maps-android/pull/1396))
+* Fix circle and heatmap layers not being aligned with globe's surface. ([#1396](https://github.com/mapbox/mapbox-maps-android/pull/1396))
 * Optimise the frequency to update location layer's visibility. ([1399](https://github.com/mapbox/mapbox-maps-android/pull/1399))
 * Fix the delay in the first appearance of the location puck. ([1403](https://github.com/mapbox/mapbox-maps-android/pull/1403))
 * Remove `android.permission.GET_TASKS` permission from the SDK. ([1430](https://github.com/mapbox/mapbox-maps-android/pull/1430))
+* Fix lag during night/day style switching when globe is enabled. ([#1432](https://github.com/mapbox/mapbox-maps-android/pull/1432))
+* Fix crash on globe symbol placement that happens during style change. ([#1432](https://github.com/mapbox/mapbox-maps-android/pull/1432))
+* Fix incorrect image source rendering with terrain enabled. ([#1432](https://github.com/mapbox/mapbox-maps-android/pull/1432))
+* Fix in-memory tile cache for non-geometry tiles when the map projection changes. Before, the tiles cached while the previous projection was active could not be used with the new projection active. ([#1432](https://github.com/mapbox/mapbox-maps-android/pull/1432))
+* Fix unexpected request delay for tiles taken from in-memory cache when minimumTileUpdateInterval is set. ([#1432](https://github.com/mapbox/mapbox-maps-android/pull/1432))
```

Comment:
This line is added at https://github.com/mapbox/mapbox-maps-android/pull/1432/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR51

Reply:
In the gl-native changelog it actually points to the single PR 3516

Reply:
Removed